pkg/ccl/backupccl/backup_test.go

```diff
@@ -8031,3 +8031,63 @@ func TestBackupOnlyPublicIndexes(t *testing.T) {
 		sqlDB.Exec(t, `DROP DATABASE restoredb CASCADE;`)
 	}
 }
+
+func TestBackupWorkerFailure(t *testing.T) {
```

Comment:
I believe this is skipped on master, do we want to pre-emptively skip it on this branch too?